kuma/settings/common.py

```diff
@@ -589,7 +589,7 @@ def _get_locales():
     {
         'NAME': 'jinja2',
         'BACKEND': 'django_jinja.backend.Jinja2',
-        'DIRS': [path('jinja2')],
+        'DIRS': [path('jinja2'), path('static')],
```

Comment:
This could be ``path('kuma/static')``, and then ``make build-static`` wouldn't be required. We can tease this apart in the next asset pipeline.